addons/crm/data/crm_lead_prediction_data.xml

```diff
@@ -38,7 +38,7 @@
         <field name="model_id" ref="model_crm_lead"/>
         <field name="state">code</field>
         <field name="code">model._cron_update_automated_probabilities()</field>
-        <field name="interval_number">1</field>
+        <field name="interval_number">7</field>
```

Comment:
Wait, no.

Reply:
I will revert, the spec in the task seems outdated for some CRON jobs and I will wait a little longer until it is merged.